argobots.spec

```
@@ -1,22 +1,33 @@
 Name: argobots
-Version: 1.0
-Release: 1%{?dist}
+
+%global major 1
+%global minor 1
+%global prerelease a1
+%global tag %{major}.%{minor}%{prerelease}
```

In the future, when there is no value in `%{prerelease}`, the above will not work.
```suggestion
%global tag %{major}.%{minor}%{?prerelease}
```

---

Oops, you are right, thanks !!